optimize/pom.xml

```diff
@@ -108,6 +109,56 @@
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
```

Comment:
this is the main fix, to move jetty dependencies to the parent dependency management. This allows us to declare the jetty version explicitly and not have spring jetty take precedence